package.json

```diff
@@ -14,11 +14,8 @@
     "lint": "eslint src stories .storybook .circleci __tests__",
     "lint-fix": "npm run lint -- --fix",
     "lint-ci": "npm run lint -- -o lint-results.json -f json",
-    "test": "jest --coverage",
-    "test-u": "jest --coverage --u",
-    "test-ci": "npm t -- --json --outputFile=test-results.json",
```

Comment:
This is required by CI and it's useful even if we've got no tests right now. You can add `--passWithNoTests` to succeed even with no tests being found.